backend/oasst_backend/utils/hugging_face.py

```diff
@@ -35,15 +35,17 @@ async def post(self, input: str) -> Any:
             inference: the inference we obtain from the model in HF
         """
 
-        session = aiohttp.ClientSession()
-        payload: Dict[str, str] = {"inputs": input}
-        response = await session.post(self.api_url, headers=self.headers, json=payload)
-
-        # If we get a bad response
-        if response.status != 200:
-            raise OasstError("Response Error Detoxify HuggingFace", error_code=OasstErrorCode.HUGGINGFACE_API_ERROR)
-
-        # Get the response from the API call
-        inference = await response.json()
+        async with aiohttp.ClientSession() as session:
+            payload: Dict[str, str] = {"inputs": input}
```

Comment:
this is still strange. If you cannot resolve the strange git merge problem, could you maybe just create a new branch on current HEAD or main and add the two files filldb.py and the filldb.md ?